src/ape_accounts/accounts.py

```diff
@@ -118,20 +118,22 @@ def sign_message(self, msg: SignableMessage) -> Optional[MessageSignature]:
             s=to_bytes(signed_msg.s),
         )
 
-    def sign_transaction(self, txn: TransactionAPI) -> Optional[TransactionSignature]:
+    def sign_transaction(self, txn: TransactionAPI, **kwargs) -> Optional[TransactionAPI]:
         user_approves = self.__autosign or click.confirm(f"{txn}\n\nSign: ")
         if not user_approves:
             return None
 
-        signed_txn = EthAccount.sign_transaction(
+        signature = EthAccount.sign_transaction(
             txn.dict(exclude_none=True, by_alias=True), self.__key
         )
-        return TransactionSignature(  # type: ignore
-            v=signed_txn.v,
-            r=to_bytes(signed_txn.r),
-            s=to_bytes(signed_txn.s),
+        txn.signature = TransactionSignature(  # type: ignore
```

Comment:
This should not longer be needed now that we have Pydantic mypy extension working!

```suggestion
        txn.signature = TransactionSignature(
```

Reply:
I tried directly and via `pre-commit`, and seems like this is still a problem (perhaps because this is a `NamedTuple` subclass?)

Reply:
Ah, noticed this was resolved in https://github.com/ApeWorX/ape/pull/1212